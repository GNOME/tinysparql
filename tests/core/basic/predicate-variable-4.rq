PREFIX ns: <http://example.org/ns#>
PREFIX x:  <http://example.org/x/>

SELECT ?s ?p WHERE { ?s ?p x:A } ORDER BY ?s ?p
