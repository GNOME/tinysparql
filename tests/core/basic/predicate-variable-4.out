"http://example.org/ns#p"	"http://www.w3.org/2000/01/rdf-schema#domain"
"http://example.org/x/p"	"http://www.w3.org/2000/01/rdf-schema#domain"
"http://example.org/x/x"	"http://www.w3.org/1999/02/22-rdf-syntax-ns#type"
